src/wast-parser.cc

```diff
@@ -616,6 +616,16 @@ Result WastParser::ParseLimits(Limits* out_limits) {
   } else {
     out_limits->has_max = false;
   }
+  if (PeekMatch(TokenType::Shared)) {
```

Comment:
I updated the spec text to show the text format for this:
https://webassembly.github.io/threads/text/types.html#memory-types

Reply:
Updated.